pkg/systemd/quadlet/quadlet.go

```diff
@@ -311,22 +312,23 @@ var (
 
 	// Supported keys in "Network" group
 	supportedNetworkKeys = map[string]bool{
-		KeyLabel:                true,
-		KeyDNS:                  true,
-		KeyContainersConfModule: true,
-		KeyGlobalArgs:           true,
-		KeyDisableDNS:           true,
-		KeyDriver:               true,
-		KeyGateway:              true,
-		KeyIPAMDriver:           true,
-		KeyIPRange:              true,
-		KeyIPv6:                 true,
-		KeyInternal:             true,
-		KeyNetworkName:          true,
-		KeyOptions:              true,
-		KeyServiceName:          true,
-		KeySubnet:               true,
-		KeyPodmanArgs:           true,
+		KeyLabel:                 true,
+		KeyDNS:                   true,
+		KeyContainersConfModule:  true,
+		KeyGlobalArgs:            true,
+		KeyDisableDNS:            true,
+		KeyDriver:                true,
+		KeyGateway:               true,
+		KeyIPAMDriver:            true,
+		KeyIPRange:               true,
+		KeyIPv6:                  true,
+		KeyInternal:              true,
+		KeyNetworkName:           true,
+		KeyNetworkDeleteWhenStop: true,
```

Comment:
Would `NetworkDeleteOnStop` sound better?

Reply:
Also I suppose since this is already part of the network section we might also be able to get rid of the `Network` prefix?

Reply:
Sure, done